views/templates/hook/widget.tpl

```diff
@@ -26,5 +26,11 @@
     s1.setAttribute('crossorigin', '*');
     s0.parentNode.insertBefore(s1, s0);
   })();
+  {if $cookie->email != ''}
+    $_Tawk_API.visitor = {
+      name  : '{$cookie->customer_firstname}',
```

Comment:
I always thought it was `$cookie->customer->firstname`. Shouldn't it be?

Also, could you escape for javascript please? Like so:
`name: '{$cookie->customer->firstname|escape:'javascript'}',`

Reply:
I cant speak to the use of $cookie->customer->firstname, as I didnt test it that way.  The way I have it I run with, and it works fine.  I recommend you try it your way, and see if it works.....

Reply:
Ahh `$cookie->customer_firstname` is the only option that works